github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -140,6 +140,39 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-kubevirt-e2e-k8s-1.19-sig-network
+  annotations:
+    testgrid-create-test-group: "false"
```

Comment:
We should include testgrid config here to show the results in https://testgrid.k8s.io/kubevirt-periodics, something like https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L309-L310